src/index.js

```diff
@@ -20,10 +20,10 @@ const VueTribute = {
   watch: {
     options: {
       handler: function (newVal) {
-        this.$nextTick(()=>{ // recreate the tribute instance for newest options
+        this.$nextTick(() => { // recreate the tribute instance for the newest options
```

Comment:
I believe most of this code can be replaced by ```this.tribute.append(0, newVal.values, true);``` it would also avoid re-creating a new instance of Tribute.